### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by chumaachike**
Created the Capstone project with all requirements

### CODE REVIEW

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻️♻️♻️ 

Hi @chumaachike,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ Correct Github workflow
✅ No linter error
✅ Design looks cool 👍

## Required Changes ♻️
_Check the comments under the review._

- [ ] You have missed out on recording a video presenting your capstone project. It's required you to record a video explaining the idea behind your project, 5 minutes long, and points out sections or the site and code section you are most proud of implementing in your project.
- [ ] Kindly add the link of your website hosted on Github Pages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          </p>
+        </div>
+        <div class="item">
+          <img src="images/icons-3.svg" class="item-icon" alt="" />
+          <h3 class="item-title">Games</h3>
+          <p class="item-description">
+            Lorem ipsum dolor sit amet consectetur adipisicing elit.
+          </p>
+        </div>
```
[the remark this answers is not in the document]
Comment (chumaachike) [reply index.html:84]: There is a predetermined gap between all elements but when squished too small the gap
Screenshot (8)

[the remark this answers is not in the document]
Comment (AmaduKamara) [reply index.html:84]: Ok, no problem. Well, you can reduce the font size of all texts on a mobile screen. 👍 Good job. Work on the rest.

**Review by ShahierNashaat — APPROVED** [review]
Hi @chumaachike , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**